src/config/dynamic-theme-fixes.config

```diff
@@ -15998,6 +15998,15 @@ paper-radio-button
 
 ================================
 
+studip.uni-passau.de
+
+CSS
+#layout_container > div {
+  background: 0;
```

Comment:
This isn't valid CSS. Maybe try:
```suggestion
  background-image: none;
```

Reply:
It actually is valid for Stud.IP itself as can be seen here (this is the standard configuration which gets overriden by Dark Reader):
![grafik](https://user-images.githubusercontent.com/12763829/167949470-3f8205e1-92c3-4532-a777-357d96679268.png)

Reply:
It's preferred to avoid such syntax.